phoenix-pherf/src/main/java/org/apache/phoenix/pherf/jmx/MonitorManager.java

```diff
@@ -135,10 +155,8 @@ public synchronized boolean shouldStop() {
         return shouldStop;
     }
 
-    public synchronized void stop() {
-        this.shouldStop = true;
-    }
-
+    // Convenience method for testing.
+    @SuppressWarnings("unused")
     public synchronized long getRowCount() {
```

Comment:
No extra synchronization needed here because rowCount is already AtomicLong. Unless the intent is to not let shouldStop, rowCount, isRunning get modified concurrently since you are guarding accesses to them by doing synchronizing on "this" .